secret_store/src/key_server_cluster/cluster.rs

```diff
@@ -1388,11 +1388,11 @@ pub mod tests {
 		let mut core = Core::new().unwrap();
 		let clusters = make_clusters(&core, 6028, 3);
 		run_clusters(&clusters);
-		loop_until(&mut core, time::Duration::from_millis(300), || clusters.iter().all(all_connections_established));
+		loop_until(&mut core, Duration::from_millis(300), || clusters.iter().all(all_connections_established));
```

Comment:
Perhaps those Durations could be moved to constants as well?

Reply:
Any naming suggestion? Something like `TIMEOUT`?